exp/runtime/internal/controllers/extensionconfig_controller.go

```diff
@@ -163,3 +201,41 @@ func discoverExtensionConfig(ctx context.Context, runtimeClient runtimeclient.Cl
 	conditions.MarkTrue(discoveredExtension, runtimev1.RuntimeExtensionDiscoveredCondition)
 	return discoveredExtension, nil
 }
+
+// reconcileCABundle reconciles the CA bundle for the ExtensionConfig.
+// cert-manager code: pkg/controller/cainjector/sources.go certificateDataSource (avoid cert-manager import via unstructured).
+func reconcileCABundle(ctx context.Context, client client.Client, config *runtimev1.ExtensionConfig) error {
+	// FIXME(sbueringer) make this more efficient if possible (not every reconcile (?), what about secret updates)
```

Comment:
```suggestion
```
I think it's good enough for now. Given that we don't have that much ExtensionConfig's and we don't reconcile them very frequently